libraries/AP_BattMonitor/AP_BattMonitor.cpp

```diff
@@ -74,7 +74,7 @@ const AP_Param::GroupInfo AP_BattMonitor::var_info[] = {
     // @Param: 2_MONITOR
     // @DisplayName: Battery monitoring
     // @Description: Controls enabling monitoring of the battery's voltage and current
-    // @Values: 0:Disabled,3:Analog Voltage Only,4:Analog Voltage and Current,5:SMBus,6:Bebop
+    // @Values: 0:Disabled,3:Analog Voltage Only,4:Analog Voltage and Current,5:SMBus,6:Bebop,7:Maxcell
```

Comment:
I think that Maxcell belongs to SBus in the interface.
I think that you can distinguish among SBus.